### ISSUE TITLE
ModuleNotFoundError: No module named 'gluon_prototype'

### ISSUE THREAD

**Issue by VedantKhairnar** [Tessellate-Imaging/monk_v1#74]
Even after being in the working directory and recloning the monk before the import, the following error is observed.
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-52-d20413af28a5> in <module>()
      1 # Using mxnet-gluon backend
----> 2 from gluon_prototype import prototype
      3 
      4 # For pytorch backend
      5 #from pytorch_prototype import prototype

ModuleNotFoundError: No module named 'gluon_prototype'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------

**Comment by abhi-kumar** [comment]
The issue has been identified with latest python 3.7.6. Use python 3.6 versions till we add a support for python 3.7.6

**Comment by vidyap-xgboost** [comment]
I've faced the same issue but it was solved when I was working on Colab. Can you @VedantKhairnar tell on which environment you're working on?

**Comment by VedantKhairnar** [comment]
@vidyap-xgboost  On the same colab notebook, earlier it was working (till I trained it on 2 ResNet models) but started facing this issue on the third model.
Sometimes restarting the kernel and doing the whole process made it work but it isnt a reliable solution.

**Comment by vidyap-xgboost** [comment]
@VedantKhairnar I agree. It happened to me once and I thought it was my mistake and I restarted again then it worked!

**Comment by VedantKhairnar** [comment]
@vidyap-xgboost  Exactly!!
